config/jwt.config.js

```diff
@@ -0,0 +1,6 @@
+const dotenv = require('dotenv');
+dotenv.config();
```

Comment:
reduce using dotenv.config() in every file that need env value by providing configfile (let say in config/index.js). That file consist of all env that are we gonna use.

example:
_config/index.js_
```
module.exports = {
  jwtSecretKey: process.env.SECRET_KEY,
  dbUser: process.env.USER,
  // and so on
}
```

_config/jwt.config.js_
```
const {
  jwtSecretKey
} = require('../config');

module.exports = {
  secret: jwtSecretKey
};
```

Reply:
sementara ini dulu men, btw `controllers/booksController.js` kebanyakan line nya. Gw ga bisa comment disana jadinya. kayanya karna relate sama yg gw komen di `controllers/borrowingController.js`

Reply:
sementara ini dulu men, btw controllers/booksController.js kebanyakan line nya. Gw ga bisa comment disana jadinya. kayanya karna relate sama yg gw komen di controllers/borrowingController.js

Reply:
semangat men nanti gw cicil lagi klo kosong hehe